src/main/resources/db/migration/V1_8__alter_contacts.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE contacts RENAME COLUMN prisonCode TO prison_code;
+ALTER TABLE contacts RENAME COLUMN prisonNumber TO prison_number;
+CREATE UNIQUE INDEX contacts_uni_idx_owner_prison_number ON contacts (owner, prison_number) WHERE prison_number IS NOT NULL
```

Comment:
We cant create a unique constraint because `null` is considered a value in postgres, so you'd only be allowed 1 contact without a prisonNumber per owner.
The documented solution is to create a partial unique index, with a where clause only including records with a prison number 👍